analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "workspace",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "workspaces": [
+      "packages/*"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "private": true,
+  "keywords": [],
+  "author": "",
```

Comment:
Remove empty "keywords" and "author", they are not required.
I think "scripts" and "main" can be removed as well for consistency with our NPM test projects. Except if they are required for Yarn workspaces to work, of course.

Reply:
Done.